pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[build-system]
+# Minimum requirements for the build system to execute
+requires = [
+    "cython>=0.28,<3.0",
+    "packaging",
```

Comment:
Are you directly using packaging somewhere for version checks?

Reply:
No don't think so I'll pull it